dev/README.md

```diff
@@ -172,6 +173,14 @@ export DRONE_TOKEN=<Your DRONE_TOKEN>
 drone sign --save grafana/oncall .drone.yml
 ```
 
+## Tracing setup
+
+Run these steps to enable tracing in your local deployment
+
+1. Add `tracing` to COMPOSE_PROFILES variable (more in [`COMPOSE_PROFILES`](#compose_profiles))
+2. Copy content of `.env.tracing.dev` to your env.dev
```

Comment:
related to comment in `.env.traces.dev`, if we took that approach, step 2 wouldn't be necessary